build/ci.go

```diff
@@ -697,9 +697,9 @@ func doDockerBuildx(cmdline []string) {
 	pass := getenvBase64("DOCKER_HUB_PASSWORD")
 
 	if len(user) > 0 && len(pass) > 0 {
-		auther := exec.Command("docker", "login", "-u", string(user), "--password-stdin")
```

Comment:
`auther`is correct here:  it's the command to perform necessary authentication for the image we are about to push.